.tidyallrc

```diff
@@ -1,3 +1,3 @@
 [PerlTidy]
 select = {lib,products,script,tests,t}/**/*.{pl,pm,t}
-argv = --pro=$ROOT/.perltidyrc
+argv = --pro=$ROOT/.perltidyrc -g
```

Comment:
I guess you can't change this as it will affect everyone.

Reply:
yes, only was for testing purposes now